### ISSUE TITLE
Not able to get the scanned data for Zebra barcode scanners

### ISSUE THREAD

**Issue by aditya-babu1** [capacitor-community/bluetooth-le#581]
**Describe the bug**
We are able to connect with the Zebra Barcode device(DS2278 model), but after scanning data transmission is not happening, does this plugin support all the devices in low energy mode?

**Plugin version:**
- @capacitor-community/bluetooth-le: 2.2.3

**Comment by pwespi** [comment]
Yes, this plugin should support all Bluetooth Low Energy devices (however not Bluetooth classic).

Do you know the Bluetooth Low Energy protocol used by this Zebra device? You will have to use `startNotifications` with a specific service and characteristic to obtain any data.

**Comment by aditya-babu1** [comment]
yes @pwespi, we tried using startNotifications with all service and characteristic ids having notify/indicate true individually, also tried enabling notifications using appropriate descriptor id and then used startNotifications with appropriate service and characteristic id, but data-transfer is not happening from zebra barcode scanner, do you see anything wrong here, can you please help on this?

**Comment by pwespi** [comment]
Do you have the specification of the Zebra device? Maybe you need to write some configuration to a characteristic to enable the data transmission.
